.travis.yml

```diff
@@ -60,7 +60,7 @@ stages:
 
 # Only building master means that we don't run two builds for each pull request.
 branches:
-  only: [master]
+  only: [master,0_5_x]
```

Comment:
could skip this part?

Reply:
Going to leave it until we publish 0.6...